.circleci/config.yml

```diff
@@ -62,8 +62,13 @@ jobs:
           environment:
             JAVA_HOME: /usr/lib/jvm/default-java
           command: |
-            make rsthtml
+            make rsthtml SPHINXOPTS="-b html -W --keep-going -n -T"
```

Comment:
notes:
rsthtml default is `-b html` but it doesn't hurt to be explicit with the full opts passing
`-W` turns warnings into errors for 'most' serious warnings (not all)
`--keep-going` is default for rsthtml, but might as well be explicit in case sphinx changes defaults (which it has)
`-n` turns nitpicky mode on to catch all types of warnings (strict mode)
`-T` is full stack trace to aid in debugging any warning or errors (helpful on the circleci build page and doesn't hurt the build time)